hack/checkout_account.py

```diff
@@ -27,24 +27,29 @@
 RESOURCE_TYPE = "aws-account"
 USER = "cluster-api-provider-aws"
 
-if __name__ == "__main__":
-    conn = httplib.HTTPConnection(BOSKOS_HOST)
-
+def post_request(input_state = "free"):
     conn.request("POST", "/acquire?%s" % urllib.urlencode({
         'type': RESOURCE_TYPE,
         'owner': USER,
-        'state': 'free',
+        'state': input_state,
         'dest': 'busy',
     })
 
     )
-    resp = conn.getresponse()
+    return conn.getresponse()
+
+if __name__ == "__main__":
+    conn = httplib.HTTPConnection(BOSKOS_HOST)
+
+    resp = post_request()
```

Comment:
You'll need to pass `conn` in, won't you?

Reply:
absolutely!